lang/python/python3/patches/021-openssl-deprecated.patch

```diff
@@ -0,0 +1,88 @@
+--- a/Modules/_ssl.c
++++ b/Modules/_ssl.c
+@@ -47,6 +47,7 @@ static PySocketModule_APIObject PySocketModule;
+ 
+ /* Include OpenSSL header files */
+ #include "openssl/rsa.h"
++#include "openssl/dh.h"
+ #include "openssl/crypto.h"
+ #include "openssl/x509.h"
+ #include "openssl/x509v3.h"
+@@ -128,13 +129,13 @@ static void _PySSLFixErrno(void) {
+ #include "_ssl_data.h"
+ 
+ #if (OPENSSL_VERSION_NUMBER >= 0x10100000L) && !defined(LIBRESSL_VERSION_NUMBER)
+-#  define OPENSSL_VERSION_1_1 1
+-#  define PY_OPENSSL_1_1_API 1
++# define OPENSSL_VERSION_1_1 1
++# define PY_OPENSSL_1_1_API 1
```

Comment:
this looks like patch-noise

Reply:
I don't know if this is valid anymore, but it made a difference back in GCC5 days. With 1 space after '#' it worked. with 2 it didn't.

Reply:
ohhh right;
thanks for the reminder

Reply:
Just tested. Makes no difference anymore. Removing.